apps/aehttp/src/ws_int_dispatch.erl

```diff
@@ -21,7 +21,8 @@ execute(Target, Action, Payload) ->
 
 -spec is_valid(Target :: atom(), Action :: atom(), Payload :: map()) ->
                 true | {false, ErrMsg :: binary}.
-is_valid(chain, get, #{<<"height">> := Height}) when Height < 0->
+is_valid(chain, get, #{<<"height">> := Height})
+    when not is_integer(Height); Height < 0 ->
```

Comment:
(Not needed for this PR) Usage of `?GENESIS_HEIGHT`?

Reply:
Old code, but yes correct.